tiatoolbox/utils/transforms.py

```diff
@@ -164,6 +164,25 @@ def bounds2size(bounds, origin="upper"):
             Defaults to upper.
     """
     left, top, right, bottom = bounds
+    origin = origin.lower()
     if origin == "upper":
         return np.array([right - left, bottom - top])
-    return np.array([right - left, top - bottom])
+    if origin == "lower":
```

Comment:
Change the return behavior to match with locsize2bounds. Ideally locsize2bound <=> bound2locsize